infra/database/mongo-database-service/MongoDatabaseService.ts

```diff
@@ -5,6 +5,9 @@ import { appConfig } from '../../../domain/config/appConfig';
 export class MongoDatabaseService {
   connectPromise: Promise<Mongoose>;
   constructor(private dbUri: string) {
+    if (!dbUri) {
```

Comment:
FWIW, I had the value set to `UNSET` for the branch and local environments for netlify - so this wouldn't have triggered.